.gitignore

```diff
@@ -11,3 +11,6 @@ oclif.manifest.json
 tsconfig.tsbuildinfo
 
 .DS_Store
+
```

Comment:
Please add .gitattributes to fix crlf / lf issue for windows b/w devcontainer/wilndows
Take it from template ;)